src/java/org/apache/cassandra/io/UnversionedSerializer.java

```diff
@@ -18,6 +18,34 @@
 
 package org.apache.cassandra.io;
 
+import java.io.IOException;
+
+import org.apache.cassandra.io.util.DataInputPlus;
+import org.apache.cassandra.io.util.DataOutputPlus;
+
 public interface UnversionedSerializer<T> extends AsymmetricUnversionedSerializer<T, T>
 {
+    static <T> UnversionedSerializer<T> singleton(T value)
```

Comment:
where is this used?

Reply:
we spoke in slack

It used to be used but after moving off the all/non bit sets, nothing uses and i forgot to cleanup